learning/tour-of-beam/frontend/README.md

```diff
@@ -0,0 +1,25 @@
+<!--
+    Licensed to the Apache Software Foundation (ASF) under one
+    or more contributor license agreements.  See the NOTICE file
+    distributed with this work for additional information
+    regarding copyright ownership.  The ASF licenses this file
+    to you under the Apache License, Version 2.0 (the
+    "License"); you may not use this file except in compliance
+    with the License.  You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing,
+    software distributed under the License is distributed on an
+    "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+    KIND, either express or implied.  See the License for the
+    specific language governing permissions and limitations
+    under the License.
+-->
+
+# Tour of Beam
+
+## Getting started
```

Comment:
Add build steps here please. More so that we use `build_runner` now.

Reply:
We need a substantially more elaborate build manual so any devops who knows nothing about Flutter can build, run, test, and otherwise verify the build without reading external manuals.

Reply:
One gotcha is that the build runner must also be ran on `playground_components` separately.

Reply:
Here is a good example of a README: https://github.com/apache/beam/tree/master/website
It takes two commands to run our website locally, and they are described.

It does not describe the build because it is handled in Jenkins. In our case we need to describe deployment, and a local run is just a simplified case of a general build.